lib/partition_setup.pm

```diff
@@ -486,9 +486,10 @@ sub select_first_hard_disk {
               if ($matched_needle && $matched_needle->{needle}->has_tag("hard-disk-dev-$device-selected"));
         }
     }
-    # Check if sda is still/already selected, if not select it
-    assert_screen [qw(select-hard-disks-one-selected hard-disk-dev-sda-not-selected)];
+    # Check if sda is still/already selected, if not select it, if none disk selected, select the first one
+    assert_screen [qw(select-hard-disks-one-selected hard-disk-dev-sda-not-selected select-hard-disks-none-selected)];
```

Comment:
Can we simply this match to only `select-hard-disks-one-selected` and `select-hard-disks-none-selected`?

Reply:
If  sda disk is not the first disk device, we should use 'select-hard-disks-one-selected'

Reply:
Updated and verified.